integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/plan/handlers/BuiltInLineageV1RelationHandler.java

```diff
@@ -0,0 +1,43 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.spark.agent.lifecycle.plan.handlers;
+
+import io.openlineage.client.OpenLineage.Dataset;
+import io.openlineage.client.utils.DatasetIdentifier;
+import io.openlineage.spark.agent.util.BuiltInPlanUtils;
+import io.openlineage.spark.api.DatasetFactory;
+import io.openlineage.spark.api.OpenLineageContext;
+import io.openlineage.spark.builtin.scala.v1.LineageRelation;
+import java.util.Collections;
+import java.util.List;
+import org.apache.spark.sql.execution.datasources.LogicalRelation;
+
+public class BuiltInLineageV1RelationHandler<D extends Dataset> {
```

Comment:
Maybe `ExtensionProvidedLineageV1RelationHandler`

Reply:
Would it make sense to replace it everywhere: `builtin` -> `extension` ?

Reply:
yes